pkg/ccl/changefeedccl/show_changefeed_jobs_test.go

```diff
@@ -590,16 +589,6 @@ func TestShowChangefeedJobsAuthorization(t *testing.T) {
 		asUser(t, f, `regularUser`, func(userDB *sqlutils.SQLRunner) {
 			userDB.CheckQueryResults(t, `SELECT job_id FROM [SHOW CHANGEFEED JOBS]`, [][]string{})
 		})
-
```

Comment:
^ so jobController can also manage any changefeed job? can you add that to the release note (it just says owners or admins)